app/assets/javascripts/modules/Modules.Autocomplete.js

```diff
@@ -182,6 +183,73 @@ moj.Modules.Autocomplete = {
       return obj;
 
     }).get();
-  }
+  },
+
+  // TODO: this should be related to the autocomplete
+  // module currently being used. Right now don't really
+  // know how to do that easily :(
+  suggestionTemplate: function(data) {
```

Comment:
Should probably move this to an _handlebars_ template or something like it, just didn't have the time to do it yet 😞 .